elements/pfe-navigation/src/pfe-navigation-item.scss

```diff
@@ -34,7 +35,7 @@ $variables: (
   @media screen and (max-width: #{pfe-breakpoint(xl)}) {
     width: 100%;
   }
-  @media screen and (min-width: 500px) {
+  @media screen and (min-width: pfe-breakpoint(sm-mobile, $map: false)) { 
```

Comment:
should this have quotes?
` @media screen and (min-width: pfe-breakpoint("sm-mobile", $map: false)) { `